robot/web/web_test.go

```diff
@@ -254,12 +254,15 @@ func TestWebWithAuth(t *testing.T) {
 					entityName = options.LocalFQDN
 				}
 
+				// TODO(RSDK-4473) Reenable WebRTC when we figure out why multiple
+				// WebRTC connections across unix sockets can create deadlock in CI.
 				conn, err := rgrpc.Dial(context.Background(), addr, logger,
 					rpc.WithAllowInsecureWithCredentialsDowngrade(),
 					rpc.WithEntityCredentials(entityName, rpc.Credentials{
 						Type:    rpc.CredentialsTypeAPIKey,
 						Payload: apiKey,
 					}),
+					rpc.WithForceDirectGRPC(),
```

Comment:
There a few `Dial` calls before this one in `TestWebWithAuth` that expect an authentication error from WebRTC, so we cannot force a direct gRPC connection for those calls. Hangs in `TestWebWithAuth` seem to come from connections made with these dials that are used for arm clients, though.

Reply:
It might be good even outside of the context of this change to call out that in those tests you mentioned. I don't think it's obvious that authentication errors can only happen with webrtc.

Reply:
That's a fair point; I'm not 100% sure why the auth error only occurs with WebRTC and not direct gRPC, so I may leave no extra comments for now.